front/app/components/admin/DatePickers/DateSinglePicker/Calendar/index.tsx

```diff
@@ -46,7 +46,7 @@ const Calendar = ({
         locale={getLocale(locale)}
         startMonth={startMonth}
         endMonth={endMonth}
-        defaultMonth={defaultMonth}
+        defaultMonth={defaultMonth ?? selectedDate}
```

Comment:
Fallback to selected date unless default month is selected

Reply:
I think this behavior is pretty logical. Before, the behavior was "If the user provided a `defaultMonth` prop, use that as the default month. Otherwise use the current month". Now the behavior is "If the user provided a `defaultMonth` prop, use that as the default month. Otherwise, if there is a selected date, use that. If there is no selected date, use the current month"

Reply:
We could remove the `defaultMonth` I guess but this way it's consistent with the other date pickers

Reply:
@luucvanderzee If there's a `selectedDate`, it should get precedence over the `defaultMonth` prop. The month of the selected date is the only one that makes sense in such a case, right? Default month, and if no default month the current month, are reasonable fallbacks then, indeed.

So this should still be changed to `defaultMonth={selectedDate || defaultMonth}` then.

Reply:
> If there's a selectedDate, it should get precedence over the defaultMonth prop

I don't agree

> The month of the selected date is the only one that makes sense in such a case, right?

Not necessarily, it's possible that there are cases where you want to overwrite this.

Reply:
🤔 What would be an example of a use case where it makes sense for a `defaultMonth` to overwrite/get precedence over the selected date?

Reply:
Maybe a situation where the current date is invalid and you need to pick a different date that's in the future. It's hard to come up with an example but I'm sure they exist. The author of the library has anticipated such situations, because otherwise they wouldn't have implemented a `defaultMonth` prop in this way. If there were no situations where it would make sense to have a `defaultMonth` different from the selected month, the behavior of their date picker would be as you describe it to be. But it's not.

Reply:
I can actually think of a use case: if you have a construction like we have now for the dashboard dates, it can make sense to suggest at least a 3-month interval or something, based on your from date. So if someone selects a date in the from field, it can make sense to show 3 months in the future by default for the 'to' field.

That said, I'd prefer to actually come across the need before implementing this, but I can live with keeping it for now. Partially because I think the defaultMonth prop will be barely used in the single date picker, if at all. And we can adjust it if we find the current solution doesn't match our needs. I just hope it won't cause bugs we could have prevented.